worker/src/main/java/gov/cms/ab2d/worker/bfdhealthcheck/BFDHealthCheck.java

```diff
@@ -27,11 +29,11 @@
 
     // Nothing else should be calling this component except for the scheduler, so keep
     // state here
-    private int consecutiveSuccesses = 0;
+    private Map<Versions.FhirVersions, Integer> consecutiveSuccesses = new HashMap<>();
```

Comment:
Is there really any difference between one FHIR version being up and another FHIR version? BFD builds them in the same JAR and runs them as a single monolith.

Reply:
That may not always be true.